scripts/create-agent.ts

```diff
@@ -521,16 +521,22 @@ GITHUB_TOKEN=${envVars.GITHUB_TOKEN || process.env.GITHUB_TOKEN || ''}`;
 
   private async waitForInstanceHealthy(instanceId: string): Promise<void> {
     console.log("Waiting for agent to be healthy...");
+    const overallStartTime = Date.now();
 
     const maxAttempts = 20;
     let attempts = 0;
 
     while (attempts < maxAttempts) {
```

Comment:
I understand what's happening. Let's change the focus of this PR to no longer be about registering the timing to now just removing this outer poll altogether.

The method we import here from script utils is already doing polling so if possible we should just call that health check method